otwcode/otwarchive
# AO3-6613 Clarify GPL version

# Pull Request Checklist

* [x] (n/a) Have you added [tests for any changed functionality](https://github.com/otwcode/otwarchive/wiki/Automated-Testing)?
  as the *first* thing in your pull request title (e.g. `AO3-1234 Fix thing`)
  until they are reviewed and merged before creating new pull requests.

## Issue

https://otwarchive.atlassian.net/browse/AO3-6613

## Purpose

Resolve AO3-6613, clarify GPL version in rails layout, static pages, and readme.

## Testing Instructions

How can the Archive's QA team verify that this is working as you intended?

View and confirm all pages have been updated as per [AO3-6613](https://otwarchive.atlassian.net/browse/AO3-6613)

## References

Are there other relevant issues/pull requests/mailing list discussions?

## Credit

What name and pronouns should we use to credit you in the [Archive of Our Own's Release Notes](https://archiveofourown.org/admin_posts?tag=1)?

Saphron (denerose) - she/her.

[Jira account](https://otwarchive.atlassian.net/jira/people/70121%3A0e3bd524-5229-4635-b90e-f45ba108beb7)


[AO3-6613]: https://otwarchive.atlassian.net/browse/AO3-6613?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
